### DISCUSSION TITLE
Announcements

### DISCUSSION THREAD

**Discussion by keunes** [WeblateOrg/weblate#8962] [Q&A]
Hello,

I have 2 questions:

1. How do I (Is it possible to) give certain teams the ability to send announcements? In the [List of privileges and built-in roles](https://docs.weblate.org/en/latest/admin/access.html#list-of-privileges-and-built-in-roles) I _do_ find the permission "Manage announcements" but it seems to be a site-wide privilege. However, I (as a project admin) can send announcements, so I wonder, is it the Admin (or another role) that includes this permission? (If it is possible through a specific role, it'd be good to mention this in the linked documentation.)
2. In the documentation on [Announcements](https://docs.weblate.org/en/latest/admin/announcements.html) it is noted that announcements can be sent "site-wide, per project, component, or language". However, from the screenshots included there it seems that's (currently) only possible via Django. Is it correct that this functionality is not (yet) exposed in the main interface?

**Comment by nijel** [comment]
1. Currently, posting announcements is limited to component/project admins.
2. AFAIR only the language wide announcements are currently not available from Weblate interface, all others are (Manage / Post announcement).

**Reply by keunes** [reply]
> Currently, posting announcements is limited to component/project admins

Ok, I have created a feature request to have such permission: #8974
(side note: I'm not familiar with the concept of 'component admins' - at least I didn't find anything in the interface or the documentation.)

> AFAIR only the language wide announcements are currently not available from Weblate interface, all others are (Manage / Post announcement).

I don't currently see an option to select the component or language (on hosted Weblate, that is). I have created a feature request: #8973

**Reply by nijel** [reply]
component/project admins: user with "Component edit"/"Project edit" permission
